src/aim/pytorch/models.py

```diff
@@ -3,14 +3,16 @@
 # --------------------------------------------------
 # References:
 # https://github.com/facebookresearch/ImageBind
-from typing import Any, Callable, List, Optional, Tuple, Union
+from typing import Any, Callable, List, Optional, Tuple, Union, Dict
 
 import torch
 from torch import nn
 
-from .. import mixins
```

Comment:
Is there a reason for changing `from . import layers` to `import layers`? If not, prefer the latter.